### PR TITLE
last one

### PR DESCRIPTION
**Pull request by nikita-kubarkov**
finish

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,19 +5,16 @@
     <title></title>
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet">
```
Comment (andrei-kubarkov) [inline index.html]: What does this tag do? Seems not needed here.

File: JS/index.js [hunk 2]
```diff
@@ -16,19 +16,27 @@ let waves = 1000;
...
+let runNum = 1;
+let enemyNum = 1;
 
+
+
+function getBattlefield(){
+    return  document.getElementById("outside-spaceship")
 }
 
 async function newGame() {
```
Comment (andrei-kubarkov) [inline JS/index.js]: You did not address the issue where I've mentioned this function.

File: JS/index.js [hunk 3]
```diff
@@ -200,23 +216,27 @@ function isGameOver() {
...
             }
+            let enemy = addEnemy(timez);
+            moveEnemyDown(enemy);
+            await new Promise(r => setTimeout(r, waves));
         }
     }
 }
 
 async function createDoubleWaves() {
     if (GAME_STATE === GAME_STATE_ACTIVE) {
```
Comment (andrei-kubarkov) [inline JS/index.js]: You validate game state twice here and this makes no sense. Remove external "if"

File: JS/index.js [hunk 4]
```diff
@@ -200,23 +216,27 @@ function isGameOver() {
 async function createWaves() {
     for (let i = 0; GAME_STATE === GAME_STATE_ACTIVE && i < 10; i++) {
         for (let times = 0; GAME_STATE === GAME_STATE_ACTIVE && times < 4; times++) {
```
Comment (andrei-kubarkov) [inline JS/index.js]: You validate game state in "for" cycle setup and and then also check "isGameOver" in "if" within a cycle.
It makes not sense, leaving just "if" is enough.

**Review by andrei-kubarkov — CHANGES_REQUESTED** [review]
See comments

File: index.html (hunk 1 above)
Comment (nikita-kubarkov) [reply index.html]: fin

File: JS/index.js (hunk 2 above)
Comment (nikita-kubarkov) [reply JS/index.js]: what do you mean?

File: JS/index.js (hunk 3 above)
Comment (nikita-kubarkov) [reply JS/index.js]: fin

File: JS/index.js (hunk 4 above)
Comment (nikita-kubarkov) [reply JS/index.js]: fin

**Pull request merged**